cms/bundles/templates/bundles/notification_emails/html_variant/bundle_in_review_email.html

```diff
@@ -1,16 +1,30 @@
-Hello,
+{% extends "bundles/notification_emails/html_variant/base_email.html" %}
 
-The preview of your release {{bundle_name}} is available for review.
-Please follow this link: {{bundle_inspect_url}} to review.
-Login using your existing Florence user name and password.
-Guidance on how to use Wagtail is available here [link to SharePoint TBC].
+{% block title %}Bundle "{{bundle.name}}" is ready for review{% endblock %}
 
-Changes to your release should be requested via your Tracker change log.
+{% block content %}
+    <p>
+        Hello,
+        <br>
+        <br>
+        The preview of your release {{bundle_name}} is available for review.
+        Please follow this link: {{bundle_inspect_url}} to review.
```

Comment:
You need to wrap the `bundle_inspect_url` in an anchor tag `<a href="...">...</a>` to ensure it renders as a link.